src/App/screens/GetPublished/index.js

```diff
@@ -1,8 +1,7 @@
 import React from 'react'
```

Comment:
`GetPublished` is a single screen in the `Instructor` module not at the top level (sibling of InstructorOverview, etc). The intent of the top level was to allow for multiple roles or activities (Instructor, Reviewer, Admin). The other two appear to be gone completely. If we are changing this approach I'd like to discuss.